shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1698,6 +1698,12 @@ void WebContents::DidFinishNavigation(
            frame_process_id, frame_routing_id);
     }
   }
+  content::NavigationEntry* entry = navigation_handle->GetNavigationEntry();
+
+  // If a history entry has been made and the forward/back call has been made,
+  // proceed with setting the new title
```

Comment:
Can you add a link to the Chromium bug?

Reply:
Yes, just added it to the latest commit.